components/org.wso2.carbon.identity.data.publisher.authentication.audit/pom.xml

```diff
@@ -102,8 +110,10 @@
                             org.osgi.framework; version="${osgi.framework.imp.pkg.version.range}",
                             org.osgi.service.component.*; version="${osgi.service.component.imp.pkg.version.range}",
                             org.wso2.carbon; version="${carbon.kernel.package.import.version.range}",
+                            org.wso2.carbon.user.core.util.*; version="${carbon.kernel.package.import.version.range}",
```

Comment:
is this needed?

Reply:
getInitatorId(String username, String tenantDomain) was moved into IdentityUtils as suggested. That's why this dependency is needed.